src/main/java/app/model/agents/AgentImp.java

```diff
@@ -117,26 +118,26 @@ public void draw(GraphicsContext gc)
         gc.strokeOval(x , y, radius, radius);
     }
 
-    public boolean agentDetected(Type agentType)
+    public boolean typeDetected(Type ype)
     {
         for(Ray r : view)
         {
-            if(r.getType()==agentType)
+            if(r.getType() == type)
             {
                 return true;
             }
         }
         return false;
     }
 
-    public Vector closestAgent(Type agentType)
+    public Vector closestTypePos(Type Type)
     {
         Vector closestAgentPos = null;
         double closestAgentDist = 10^5;
 
         for(Ray r : view)
         {
-            if(r.getType()==agentType)
+            if(r.getType()==type)
```

Comment:
Don't think you need to change these, they were fine as they were. Plus typo with 'ype' instead of 'type'

Reply:
To be fair, makes sense to change. Also type with the capital T in the second method.

Reply:
spelling is true, but since there is also an Enum AgentType, this seemed better.

Especially since we also use if for Type.Target in targetAgent, which isn't an agent